src/main/java/edu/harvard/iq/dataverse/harvest/server/web/servlet/OAIServlet.java

```diff
@@ -205,7 +205,7 @@ private RepositoryConfiguration createRepositoryConfiguration() {
                 .withAdminEmail(systemEmailLabel)
                 .withCompression("gzip")
                 .withCompression("deflate")
-                .withGranularity(Granularity.Second)
+                .withGranularity(Granularity.Lenient)
```

Comment:
What does this do? Was this change intentional?

Reply:
It is the single actual change in the PR, it's mentioned under "how to test". 

It's to address a small issue @kcondon found the other day; not a huge thing, we were just a little too strict with parameter formatting and it was against the OAI spec.

Reply:
Ok, the date thing. Thanks!